platform/web/js/libs/library_godot_audio.js

```diff
@@ -512,8 +531,11 @@ class SampleNode {
 	 * @returns {void}
 	 */
 	start() {
-		this._resetSourceStartTime();
-		this._source.start(this.startTime, this.offset);
+		if (!this.isStarted) {
+			this._resetSourceStartTime();
+			this._source.start(this.startTime, this.offset);
+			this.isStarted = true;
+		}
```

Comment:
Always use early return if possible.

```suggestion
		if (this.isStarted) {
			return;
		}
		this._resetSourceStartTime();
		this._source.start(this.startTime, this.offset);
		this.isStarted = true;
```